src/less/forms.less

```diff
@@ -981,20 +991,49 @@ input[type='checkbox']:focus {
 .rhinoselect {
   position: relative;
   vertical-align: middle;
+  min-height: 4rem;
 
   .mixin-chevron-down();  // add chevron
 }
 
+.select__label {
+  top: 0;
+  left: 0;
+  position: absolute;
+  border: #ccc solid 1px;
+  padding: 8px;
+  width: 100%;
+  font-weight: normal;
+  background-color: #fcfcfc;
```

Comment:
use consistent color as used in other places for background and border(use variable).